catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/catalog/CatalogApplication.java

```diff
@@ -290,11 +295,14 @@ private String getDocument(
       res.status(HttpStatus.SC_OK);
       res.type(content.getMimeTypeValue());
 
-      int bytesCopied = IOUtils.copy(content.getInputStream(), res.raw().getOutputStream());
-      res.raw().setContentLength(bytesCopied);
-      res.raw()
-          .flushBuffer(); // Flashing buffer so Spark won't set the body based on the return value
+      try (InputStream inputStream = content.getInputStream()) {
+        int bytesCopied = IOUtils.copy(inputStream, res.raw().getOutputStream());
+        res.raw().setContentLength(bytesCopied);
+        res.raw()
+            .flushBuffer(); // Flashing buffer so Spark won't set the body based on the return value
 
+        res.raw().getOutputStream().close();
```

Comment:
Do you need this line?  Doesn't `res.raw().getOutputStream()` return a new output stream?

Reply:
I don't think we'd want to close it anyways, even if it was. This should end up being the stream result sent to the other client iirc. If you close it, then they'll get a truncated result when the server closes it on its end.

Reply:
@willwill96 I just ran a test on it and two output streams from getOutputStream are equal so it should get the same one here.
@stustison I thought that flushing empties the stream so by the time the output stream gets closed it should be written out. Is there more happening here?

Reply:
The flash makes sure that the Output Stream content is returned instead of the returned String on line 332.

Reply:
Does that mean the Output Stream content or the stream is returned?

Reply:
Upon investigating closing the stream here looses headers so expect this to be removed soon

Reply:
@Kjames5269: We should probably port this change to 2.13.x, since it's already been merged with having this output stream being closed.